### ISSUE TITLE
Technical Blog: "How to deploy Doom using Zarf" 

### ISSUE THREAD

**Issue by mgrim2** [defenseunicorns/zarf#231]
Outcome: We want to provide a well-written technical blog that gives step-by-step instructions on how to deploy Doom using Zarf. 

Background: We believe that by providing a technical blog, our target user persona (i.e. junior engineers) can see the problem, solve the problem, and understand that Zarf is a tool that makes delivering software in an air gapped environment easy. 

Acceptance Criteria / Definition of Done:
-Target persona understands the problem Zarf is trying to solve 
-Target persona is able to deploy Doom using Zarf 
-Target persona understands/ gains value from the blog 

This is blocked or made more difficult by a few things:
- Need to make sure the technical content is 100% accurate. 

Target finish date: January 17th

**Comment by RothAndrew** [comment]
IMO [this page](https://github.com/defenseunicorns/zarf/tree/master/examples/game) already has (or should have) this content. Is it worth having a separate technical blog here? Can we make this page in the repo more "blogg-y" if the need isn't being met? I believe this will double the maintenance burden on keeping things up to date since we'll now need to update the Zarf docs linked above as well as the technical blog as Zarf changes.

/cc @jeff-mccoy

**Comment by gentrco** [comment]
I thought the GitHub page worked great.  If the target audience is junior engineers, I don't know why you'd need a separate tech blog.  I'm not a software engineer, but I made my way through it.  One thing I did notice, however, was that I was unable to package and deploy the game without manually copying the game files to my working directory.  When I did the "cd" step, my zarf commands wouldn't work.  As I said, I'm not a software engineer, so this could easily be user error but thought it worth a mention.
